docs/decisions/0008-secret-key-usage.rst

```diff
@@ -0,0 +1,52 @@
+Django Secret Key Usage
+-----------------------
+
+Status
+======
+
+Pending
+
+Context
+=======
+
+Django's Secret key is used by django to cryptographically sign various
+capabilities in django.  It is also used by the edx-platform in various cases
+as a `pepper`_ when hashing data to anonymize it while keeping it consistent.
+Some of the uses of the ``SECRET_KEY`` in edx-platform were not resilient to
+the key being rotated.  As a part of ARCHBOM-1646 those code paths are being
+updated so that regular rotation of the ``SECRET_KEY`` will cause temporary
+issues but will not fully break things.  ie. Sessions might get invalidated or
+data might not be correlatable over the change boundary but the change is
+otherwise safe.
+
+.. _pepper: https://en.wikipedia.org/wiki/Pepper_(cryptography)
+
+Decisions
+=========
+
+1. Any uses of ``SECRET_KEY`` should be resilient to that key being rotated.
+
+2. Wherever the ``SECRET_KEY`` is used, it should also document the impact of
+   rotating the key.
+
+For example:
```

Comment:
rST nit: This ends up styled as a heading while the other "for example" does not. Maybe something to do with line breaks and colons.